apps/codeforafrica/src/pages/stories/[slug].page.js

```diff
@@ -1,21 +1,30 @@
 import { Section } from "@commons-ui/core";
 import Box from "@mui/material/Box";
-import Typography from "@mui/material/Typography";
+import Image from "next/image";
 import React from "react";
 
+import ArticlePage from "@/codeforafrica/components/ArticlePage";
 import Page from "@/codeforafrica/components/Page";
 import RelatedStories from "@/codeforafrica/components/RelatedStories";
-import ShareBar from "@/codeforafrica/components/ShareBar";
-import {
-  FacebookShareBarButton,
-  LinkedinShareBarButton,
-  TwitterShareBarButton,
-} from "@/codeforafrica/components/ShareBarButton";
 import { getPageStaticProps } from "@/codeforafrica/lib";
 
 function Index({ article, sections, ...props }) {
   return (
     <Page {...props}>
+      <Box
+        sx={{
+          position: "relative",
+          width: "100%",
+          height: { xs: "163px", md: "600px" },
+        }}
+      >
+        <Image
+          alt="article-featured-image"
+          src={article?.coverImage?.src}
+          layout="fill"
+          objectFit="cover"
+        />
+      </Box>
```

Comment:
Why isn't this in the `ArticlePage`?

Reply:
This is because of the `Section component. The image covers takes full screen so including this in the `Article page` or the `ArticleHeader` will center the image.

Reply:
@kilemensi Moved  the image + section to articlepage to accomodate the featured image